tsconfig.prune.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "paths": {
+      "@temporalio/*": ["packages/*"]
+    }
+  },
+  "files": [
+    "./packages/activity/src/index.ts",
+    "./packages/client/src/index.ts",
+    "./packages/common/src/index.ts",
+    "./packages/create-project/src/index.ts",
+    "./packages/meta/src/index.ts",
+    "./packages/worker/src/index.ts",
+    "./packages/worker/src/workflow/workflow-worker-thread.ts",
+    "./packages/workflow/src/index.ts"
```

Comment:
We should add `./packages/workflow/src/worker-interface.ts` too

Reply:
@JoshuaKGoldberg I want to merge this, can you please add the missing file?

Reply:
Ah, sorry, I'd missed this notification! Merging now.